ducanhpham0312/zeevision
# feat: filter for tables

### Linked issue

Closes ducanhpham0312/zeevision-private#144


### Description
- Each table now has a filter feature.
  - A main filter on top of the table header. This will filter the most commonly used field depends on the table (e.g., variable name, job key, audit log's element type etc)
  - A fully customized filter under the `Filter` button, where each column can be filter with `is`, `is not`, `contains`.
  - Each cell content in `tbody` can have its content copied. The date copied will be parsed nicely to the date filter.
- The application will open the processes page as default.


### Testing

- Start the test bench and the application. 
- The application should open processes page upon startup.
- Deploy process, create instances and test the filters.
